### DISCUSSION TITLE
Convert OBJ to GLB with Javascript

### DISCUSSION THREAD

**Discussion by rruvira9191** [google/model-viewer#1821] [Q&A] [answered]
Hi guys, 

Please could you tell me if I can use the model-viewer whit OBJ file if this possible, now I just used the model-viewer whit GLB and it works perfectly, but please an I need working whit the obj object. Thanks

**Comment by sercandemirtas** [comment]
You need glb, gltf and usdz (ios ar) file formats to use model-viewer. Obj is not supported. If you have obj format of your file you can convert it via blender or there are lots of platform to do that.

**Comment by rruvira9191** [comment]
Thanks, other option could convert the obj but I need convert it whit java code I cant us an external platform for that, thanks for your reply

**Comment by elalish** [comment, marked as answer]
Three.js does this; it can load OBJ and export GLB.

**Reply by rruvira9191** [reply]
Hi , thanks for you reply, i already use Three.js but know I have a problem. when I put the THREE.Sprite in my scene I can see the sprite whit the object 3d, but when I move the object 3d the sprite move in another direction, I need know if is possible merge the Sprite whit the object 3d by when I move the object 3d the Sprite move too in the correct position.
